pkgs/development/python-modules/torch/default.nix

```diff
@@ -202,8 +202,14 @@ let
       ]);
     "MPI cudatoolkit does not match cudaPackages.cudatoolkit" =
       MPISupport && cudaSupport && (mpi.cudatoolkit != cudaPackages.cudatoolkit);
+    # This used to be a deep package set comparison between cudaPackages and
+    # effectiveMagma.cudaPackages, but it made it evaluation too strict. This
+    # was bad e.g. for cudaPackages' `aliases.nix`, forcing warnings upon
+    # everyone using torch even if the alias was never used.
+    "Magma cudaPackages does not match cudaPackages. To apply overrides consistently, use overlays" =
+      cudaSupport && (effectiveMagma.cudaPackages.cudaVersion != cudaPackages.cudaVersion);
     "Magma cudaPackages does not match cudaPackages" =
-      cudaSupport && (effectiveMagma.cudaPackages != cudaPackages);
+      cudaSupport && (effectiveMagma.cudaPackages.cudaVersion != cudaPackages.cudaVersion);
```

Comment:
I think we got two checks now:)

Reply:
:scream_cat: Sorry. Me trying to do two things at the same time. I think it's okay now, if I didn't anger nixfmt again.